velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -751,7 +751,11 @@ class PlanBuilder {
   /// "row_number() over (order by b) as a"
   /// "row_number() over (partition by a order by b
   ///  rows between a + 10 preceding and 10 following)"
-  PlanBuilder& window(const std::vector<std::string>& windowFunctions);
+  /// @param inputsSorted If true, StreamingWindowBuild is used; otherwise
```

Comment:
perhaps, instead of a boolean, introduce a new method: streamingWindow(windowFunctions)

Reply:
@mbasmanova We previously introduced a StreamingWindow function. However, upon comparison with the regular window function, we found that the code inside both functions is identical, except for the final value of the inputsSorted boolean variable. Therefore, to simplify and streamline our codebase, we have decided to remove the StreamingWindow function. For a more comprehensive discussion and further details, please refer to [here](https://github.com/facebookincubator/velox/pull/5437#discussion_r1329539926).

Reply:
@JkSelf I'm referring to PlanBuilder API here. I suggest to keep PlanBuilder::window API as is and add a new one PlanBuilder::streamingWindow.

Reply:
@mbasmanova Added PlanBuilder::streamingWindow.